ShopAppTests/App/Main/Views/Cells/CollectionView/ProductOptionsCollectionViewCellSpec.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  ProductOptionsCollectionViewCellSpec.swift
+//  ShopAppTests
+//
+//  Created by Evgeniy Antonov on 3/21/18.
+//  Copyright © 2018 RubyGarage. All rights reserved.
+//
+
+import Quick
+import Nimble
+import ShopApp_Gateway
+
+@testable import ShopApp
+
+class ProductOptionsCollectionViewCellSpec: QuickSpec {
+    override func spec() {
+        var cell: ProductOptionsCollectionViewCell!
+        var collectionView: UICollectionView!
+        
+        beforeEach {
+            let provider = ProductOptionsCollectionProvider()
+            
+            let controllerCollectionView = UICollectionView(frame: CGRect.zero, collectionViewLayout: UICollectionViewFlowLayout())
+            controllerCollectionView.registerNibForCell(ProductOptionsCollectionViewCell.self)
+            
+            controllerCollectionView.dataSource = provider
+            controllerCollectionView.delegate = provider
+            
+            let option = ProductOption()
+            option.name = "Option name"
+            option.values = ["Option value 1", "Option value 2"]
+            provider.options = [option]
+            controllerCollectionView.reloadData()
+            
+            let indexPath = IndexPath(row: 0, section: 0)
+            let dequeuedCell: ProductOptionsCollectionViewCell = controllerCollectionView.dequeueReusableCellForIndexPath(indexPath)
+            cell = dequeuedCell
+            
+            collectionView = self.findView(withAccessibilityLabel: "collectionView", in: cell) as! UICollectionView
+        }
+        
```

Comment:
Please, test `setupCollectionView` method.

Reply:
This method is private. I added checking setup of datasource and delegate for collection view

Reply:
Okay.